aws-xray-propagator/src/main/java/io/opentelemetry/contrib/awsxray/propagator/AwsXrayPropagator.java

```diff
@@ -112,49 +108,20 @@ public <C> void inject(Context context, @Nullable C carrier, TextMapSetter<C> se
     char samplingFlag = spanContext.isSampled() ? IS_SAMPLED : NOT_SAMPLED;
     // TODO: Add OT trace state to the X-Ray trace header
 
-    StringBuilder traceHeader = new StringBuilder();
-    traceHeader
-        .append(TRACE_ID_KEY)
-        .append(KV_DELIMITER)
-        .append(xrayTraceId)
-        .append(TRACE_HEADER_DELIMITER)
-        .append(PARENT_ID_KEY)
-        .append(KV_DELIMITER)
-        .append(parentId)
-        .append(TRACE_HEADER_DELIMITER)
-        .append(SAMPLED_FLAG_KEY)
-        .append(KV_DELIMITER)
-        .append(samplingFlag);
-
-    Baggage baggage = Baggage.fromContext(context);
-    // Truncate baggage to 256 chars per X-Ray spec.
```

Comment:
I can't seem to find a spec document for the xray spec format. If there is a document, and it does talk about including baggage like this, we should likely follow the spec.

Reply:
I'm not aware of a formal specification as well, but I usually follow this reference: https://docs.aws.amazon.com/xray/latest/devguide/aws-xray.html#Tracing-header:~:text=sampling%20rules.-,Tracing%20header,-All%20requests%20are

It seems that there is no equivalent concept to baggage, at least in the reference above.

Reply:
No reference to baggage here, so maybe its right to delete it.

Reply:
refer to the comment, using key "XRAY_HEADER_ADDITIONAL_FIELDS"